src/config.ts

```diff
@@ -0,0 +1,29 @@
+import fs from 'fs'
+
+import { Config } from 'types/config'
+
+import defaultConfig from '../config.json'
+
+/**
+ * Global config object
+ *
+ * Contains all possible configs for the app
+ *
+ * When forking, update the file at `/custom/config.json`.
+ */
+
+const config: Config = defaultConfig
+
+// Load optional overwrites, if any
+try {
+  const customConfig = JSON.parse(fs.readFileSync('../custom/config.json', 'utf8'))
```

Comment:
OMG why? Just `require` it

Reply:
You are also pulling in `fs` even if you mock it with `empty` in `webpack.config`
And when you are mocking, it throws, and when it throws you can't get custom config.
So in web custom config is useless.

All is solved if you use `require`

Reply:
Just because I knew you would be bothered ;)

If `require` works, then it works.